test/e2e/common/node/pods.go

```diff
@@ -235,12 +235,12 @@ var _ = SIGDescribe("Pods", func() {
 			Spec: v1.PodSpec{
 				Containers: []v1.Container{
 					{
-						Name:  "nginx",
-						Image: imageutils.GetE2EImage(imageutils.Nginx),
+						Name:  "pause",
```

Comment:
does changing from nginx to pause change the lifecycle characteristics of the pod (w.r.t. handling shutdown signals)? same question applies in multiple places

Reply:
Good question. Both the [pause container](https://github.com/kubernetes/kubernetes/blob/4bd396115def9afd7eac7f9071ed699d61c55b27/build/pause/linux/pause.c#L55-L62) and [nginx](http://nginx.org/en/docs/control.html) handle SIGINT and SIGTERM as a fast shutdown, which I think is what containerd uses. nginx additionally handles SIGQUIT, but I don't see any references to SIGQUIT in k/k, containerd, or cri-o.

Maybe @SergeyKanzhelev or @endocrimes have additional context.

Reply:
This test doesn't cover any corner cases of shutdown so this seems ok to me 👍